src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -3,106 +3,87 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class RegistrationServiceImplTest {
-    private static final String VALID_LOGIN = "BobBlink";
-    private static final String HUGE_LOGIN = "BobBlinkBobBlinkBobBlink";
-    private static final String NULL_VALUE = null;
-    private static final int VALID_AGE = 25;
-    private static final int LESS_AGE = -5;
-    private static final int HIGH_AGE = 100;
-    private static final String VALID_PASSWORD = "123456789";
-    private static final String NOT_VALID_PASSWORD = "123";
-    private static final User NULL_USER = null;
     private static RegistrationService registrationService;
-    private static User user;
+    private User user;
 
     @BeforeAll
     static void beforeAll() {
         registrationService = new RegistrationServiceImpl();
-        user = new User();
     }
 
     @BeforeEach
     void beforeEach() {
-        user.setLogin(VALID_LOGIN);
-        user.setAge(VALID_AGE);
-        user.setPassword(VALID_PASSWORD);
+        user = new User();
+        user.setLogin("BobBlink");
+        user.setAge(25);
+        user.setPassword("123456789");
     }
 
     @Test
     void checkLoginIsNull_NotOk() {
-        user.setLogin(NULL_VALUE);
-        tryToRegisterException(user, "Login can't be NULL");
+        user.setLogin(null);
+        checkException(user, "Login can't be NULL");
     }
 
     @Test
     void checkPasswordIsNull_NotOk() {
-        user.setPassword(NULL_VALUE);
-        tryToRegisterException(user, "Password can't be NULL");
+        user.setPassword(null);
+        checkException(user, "Password can't be NULL");
     }
 
     @Test
     void checkAgeIsLessThanNormal_NotOk() {
-        user.setAge(LESS_AGE);
-        tryToRegisterException(user, "Age can't be less than 18");
-    }
-
-    @Test
-    void checkAgeIsHigherThanNormal_NotOk() {
-        user.setAge(HIGH_AGE);
-        tryToRegisterException(user, "Age can't be older than 99");
+        user.setAge(-5);
+        checkException(user, "Age can't be less than 18");
     }
 
     @Test
     void checkLoginTheSame_NotOk() {
+        Storage.people.clear();
```

Comment:
you should clear storage after each test, so move this line `Storage.people.clear();` to the method with AfterEach annotation